velox/functions/prestosql/tests/utils/FunctionBaseTest.h

```diff
@@ -318,6 +318,11 @@ class FunctionBaseTest : public testing::Test,
     return result[0];
   }
 
+  // Returns a set of signatures for the given function name and return type.
+  static std::unordered_set<const exec::FunctionSignature*> getSignatures(
```

Comment:
Why do we return a set? An std::vector should work just fine, no?

Let's document 'returnType' parameter. It may not be obvious what the value is.

Reply:
I was using set to keep similar style with FunctionBaseTest::getSignatureStrings method. Changed to use vector.

Document 'returnType' parameter as below.
> The name of expected return type defined in function signature.

Thanks.